NEP-P2P_Protocol_change.mediawiki

```diff
@@ -0,0 +1,129 @@
+<pre>
+  NEP: <to be assigned>
+  Title: P2P Protocol change
+  Author: Shargon and Belane
+  Type: Standard
+  Status: Draft
+  Created: 2018-03-07
+</pre>
+
+==Abstract==
+
+We believe that a P2P protocol optimization is necessary in order to avoid unnecessary data exchange between nodes.
+In this sense, reducing the computational consumption and, consequently, improving and speeding up the communication between nodes.
+
+==Motivation==
+
+Improve the P2P protocol to support higher loads, as well as improving the communication speed and prioritizing consensus messages. 
+In addition, we think that it is necessary to implement a compression mechanism and save all bytes that should be possible.
```

Comment:
```suggestion
```
I believe the 
> Improve the P2P protocol to support higher loads, as well as improving the communication speed and prioritizing consensus messages. 

is actually a good concise and all encompassing description. 

The "compression" line specifically highlights one optimization that leads to above description. Perhaps it fits better in the rationale section?